ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -15,7 +15,7 @@ extension RACScheduler: DateSchedulerType {
 	}
 
 	public func schedule(action: () -> ()) -> Disposable? {
-		return self.schedule(action)
+		return (self.schedule(action) as RACDisposable) as Disposable?
```

Comment:
I think it might be slightly clearer what you're doing if you were to separate this into two lines and add a comment like:

```swift
let disposable: RACDisposable = self.schedule(action) // Call the Objective-C implementation
return disposable as Disposable?
```

Now that I look at it… why is this method necessary? If someone calls the original Obj-C implementation and gets a `RACDisposable` back, can't that automatically be assigned to a `Disposable?` l-val as is happening here?